Plugin/Sales/Order/Email/Container/OrderIdentityPlugin.php

```diff
@@ -32,9 +39,11 @@ public function aroundIsEnabled(\Magento\Sales\Model\Order\Email\Container\Order
         $forceOrderMailSentOnSuccess = $this->checkoutSession->getRazorpayMailSentOnSuccess();
         if(isset($forceOrderMailSentOnSuccess))
         {
-            $returnValue = $forceOrderMailSentOnSuccess;
-            $this->checkoutSession->unsRazorpayMailSentOnSuccess();           
+            // Send order confirmation email after payment completed successfully
+            $returnValue = (bool)$forceOrderMailSentOnSuccess;
```

Comment:
@enishant , this is not required. 
Just removing unset session will solve the issue.

Reply:
@ramth05 Updated changes